docs/reference/asset-reference.md

```diff
@@ -2,8 +2,8 @@
 
 ## Asset object
 
-The Asset class enables access to the UP42 assets. Assets are results of orders
-or results of jobs with download blocks.
+The Asset class enables access to the UP42 assets in the user storage. Assets are results 
```

Comment:
User Storage is an incorrect name, and can lead to confusion. The storage lives in the workspace, not on the user level. Throughout the docs I used Storage or UP42 Storage as a standard since that is also what you see in the console.

Reply:
Thanks for clarification, removed "user".